theme_bewise/views/customizations.xml

```diff
@@ -421,4 +421,24 @@
     </xpath>
 </template>
 
+<!-- ======== CTA Box ======== -->
+<template id="s_cta_box" inherit_id="website.s_cta_box">
+    <!-- Title -->
+    <xpath expr="//h2" position="replace" mode="inner">
+        Your next university<br/>is just clicks away
```

Comment:
I think the way I wrote it was confusing, but I just meant to add a "s" at the end of "click" :slightly_smiling_face: It was better with "a few" before :+1: 
(Sorry, I am used to quote like that for commit messages, so the dev knows where the text is, but it was useless here)

Reply:
Ah yes sorry, my bad 😅